package.json

```diff
@@ -13,7 +13,7 @@
     "lint:js": "eslint \"**/*.js\" --ignore-pattern node_modules/",
     "lint:css": "sass-lint 'app/**/*.scss' -v -q",
     "nuke": "npm run clean && rm -rf node_modules/",
-    "start:dev": "cross-env LOGGER_LEVEL=info babel-node server",
+    "start:dev": "cross-env LOGGER_LEVEL=info API_HOST=http://localhost:57772 DOCUMENT_HOST=http://localhost:63600 babel-node server",
```

Comment:
Any reasons for these? Are they left overs from when you was testing the full stack?

Reply:
Unfortunately, the only way to run and test document download is to run the apps locally directly and not via docker until the docker work is done on the document api. I deliberately left these in, so that those working on the front end would have a route to connect up to the other services. I can remove if you prefer.

Reply:
ah ok I see what you mean. This will not work for me or Ben at the moment since we don't run the API locally. Is it ok to remove these for now